src/sonic-yang-models/yang-models/sonic-nat.yang

```diff
@@ -0,0 +1,316 @@
+                }   
+
+                leaf local_port {
+                    description "Local port for the Static NAPT entry.";
+
+                    mandatory true;
+                    type inet:port-number ;
+                }
+                
+                leaf nat_type {
+                    description "Nat type for the static napt entry - snat or dnat";
+
+                    type nat-type;
+                    default dnat;
+                }   
+                
+                leaf twice_nat_id {
+                    description "Twice nat id for the static napt to achieve the twice napt";
+
+                    type uint16 { 
+                        range "1..9999" {
+                            error-message "Invalid twice nat id for the static NAPT.";
+                            error-app-tag twice-nat-id-invalid;
+                        }   
+                    }   
+                }
+            }
+        }
+
+        container STATIC_NAT {
+
+            description "STATIC_NAT table part of config_db.json";
+
+            list STATIC_NAT_LIST {
+                key "global_ip";
+
+                leaf global_ip {
+                    description "Global ip for the Static NAT entry.";
+
+                    type inet:ipv4-address;
+                }
+
+                leaf local_ip {
+                    description "Local ip for the Static NAT entry.";
+
+                    mandatory true;
+                    type inet:ipv4-address;
+                }
+
+                leaf nat_type {
+                    description "Nat type for the static nat entry - snat or dnat";
+
+                    type nat-type;
+                    default dnat;
+                }
+
+                leaf twice_nat_id {
+                    description "Twice nat id for the static nat to achieve the twice nat";
+
+                    type uint16 { 
+                        range "1..9999" {
+                            error-message "Invalid twice nat id for the static NAT.";
+                            error-app-tag twice-nat-id-invalid; 
+                        }   
+                    }   
+                }
+            }
+        }
+
+        container NAT_GLOBAL {
+
+            description "NAT_GLOBAL table part of config_db.json";
+
+            list NAT_GLOBAL_LIST {
+                key "name";
+                max-elements 1;
+
+                leaf name {
+                    description "Key - Name of the NAT Global table - Values.";
+
+                    type enumeration {
+                        enum Values;
+                    }
+                }
+
+                leaf admin_mode {
+                    description "Admin mode of the NAT feature.";
+
+                    type stypes:admin_mode;
+                    default enabled;
+                }
+
+                leaf nat_timeout {
+                    description "Timeout for the nat entries within the range of 300 sec to 432000 secs.";
+
```

Comment:
Can we specify default values for both TCP and UDP timeout?

Reply:
Done, specified default values for timeout.